shotover-proxy/src/message/connection_state.rs

```diff
@@ -0,0 +1,11 @@
+#[derive(Default, PartialEq, Clone, Debug)]
+pub struct CassandraConnectionState {
+    pub used_keyspace: Option<String>,
+}
+
+#[derive(PartialEq, Clone, Debug)]
+pub enum ConnectionState {
```

Comment:
These are here to show an example of how we would add further variants of `ConnectionState`